src/h3lib/include/h3api.h.in

```diff
@@ -348,6 +348,16 @@ DECLSPEC H3Error H3_EXPORT(cellsToLinkedMultiPolygon)(const H3Index *h3Set,
 DECLSPEC void H3_EXPORT(destroyLinkedMultiPolygon)(LinkedGeoPolygon *polygon);
 /** @} */
 
+/** @defgroup geoLoopAreaRads2 geoLoopAreaRads2
+ * Functions for geoLoopAreaRads2
+ * @{
+ */
+/**
+ * @brief Area in radians^2 enclosed by vertices of a GeoLoop.
+ */
+DECLSPEC H3Error H3_EXPORT(geoLoopAreaRads2)(GeoLoop loop, double *out);
+/** @} */
+
```

Comment:
let's discuss if we want to expose this in h3api.h.in just yet. We should consider how it will work in each binding too.